data-filter.html

```diff
@@ -829,6 +829,9 @@ <h3 style="text-align: center;">Selected Measurements</h3>
             }
 
             _showCardSelectAll(_selected, tableData) {
+                if (this.tableData && this.tableData.length === 0) {
+                    return true;
+                }
                 if (this.tableData && this._selected && Object.keys(this._selected).length == this.tableData.length) {
```

Comment:
just added my logic on to of that - not sure if we need to delete it, maybe there was some reason to ensure that there is exact amount of selected MS that all tableData has.